### DISCUSSION TITLE
Config error file

### DISCUSSION THREAD

**Discussion by Kristallik355** [end-4/dots-hyprland#508] [Q&A]
Config error in file /home/user/.config/hypr/hyprland/colors.conf at line 9:rgba()expects length of 8 characters (4 bytes) or 4 comma separated values

Config error in file /home/user/hypr/hyprland.conf at line 9:Config error in file /home/user/.config/hypr/hyprland/colors.conf at line 9:rgba()expects length of 8 characters (4 bytes) or 4 comma separated values

File:Colors.conf
![Screenshot_2024-05-18-13-53-14-638_com.rhmsoft.edit.jpg](https://github.com/end-4/dots-hyprland/assets/150447775/866054ed-df06-4562-9148-d7b26cead921)

![Screenshot_2024-05-18-13-53-32-400_com.rhmsoft.edit.jpg](https://github.com/end-4/dots-hyprland/assets/150447775/1ef57f6e-a47c-43a0-aaa9-e9b644866dd0)

File:hyprland.conf
![Screenshot_2024-05-18-13-52-50-972_com.rhmsoft.edit.jpg](https://github.com/end-4/dots-hyprland/assets/150447775/d7f75c7f-5bc9-43f0-9646-c442c33159e3)

**Comment by end-4** [comment]
- did you install with the script? were all package installations successful?
- can you run `sass ~/.config/ags/scss/main.scss ~/.cache/ags/user/generated/style.css` and show the output?

**Reply by Kristallik355** [reply]
I installed using this method( bash <(curl -s "https://end-4.github.io/dots-hyprland-wiki/setup.sh") ).When I installed there were no problems except a few warnings for Python files but they were at the end of the installation

The result:![IMG_20240519_182751.jpg](https://github.com/end-4/dots-hyprland/assets/150447775/671c8713-bece-45a2-9e55-48145c89c6dd)

**Reply by end-4** [reply]
have you picked a wallpaper? Ctrl+Super+T

**Reply by Kristallik355** [reply]
No,i turned it on transparency

**Reply by end-4** [reply]
not sure i get it

**Reply by Kristallik355** [reply]
Could it be a dual boot problem?
I use windows 10 home and archlinux and I installed these OS on my laptop
Laptop Specifications:
Laptop Name:HP Pavilion 15 Notebook-pc
Processor:Intel Core i3 4030U
RAM:8GB DDR3
Graphics:NVIDIA GeForce 830M
Memory:1TB HDD

**Reply by end-4** [reply]
unrelated

**Reply by Kristallik355** [reply]
Okay, let's skip this problem last time I used your version of hyprland and the problem in it was in the wallpaper changes
The problem was exactly the same
